source/framework/CMakeLists.txt

```diff
@@ -11,10 +11,6 @@ file(GLOB_RECURSE addon_src
         "tiny*cpp"
         "startup.cpp")
 
-if (NOT ${REST_EVE} MATCHES "ON")
```

Comment:
why this is removed?

Reply:
It is already added in the main CMakeLists.txt